source/_statement.html.md

```diff
@@ -33,14 +33,28 @@ The content listed below is non-accessible for the following reasons.
 
 ### Non compliance with the accessibility regulations
 
-The side navigation on the administrative platform sometimes does not accurately show where a user is on the website. This fails WCAG 2.1 AA success criterion 2.4.7 (focus visible).
+We have identified the following issues:
 
-Some menu entries do not appear in an active state when a user browses their content. This fails WCAG 2.1 AA success criterion 3.2.3 (consistent navigation).
+On our administrative portal, not all of our pages are identified with a unique title.
 
-We'll make all these elements fully accessible by September 2020.
+We'll make all these elements fully accessible by the end of the year.
```

Comment:
By "these elements" are we referring to line 38 about the admin portal? If so, it should be singular. So just "We'll make this fully accessible by the end of the year."